dot_parser.py

```diff
@@ -548,8 +548,13 @@ def parse_dot_data(s):
         tokens = graphparser.parseString(s)
         return list(tokens)
     except ParseException as err:
-        print(
-            err.line +
-            " "*(err.column-1) + "^" +
-            err)
+        if PY3 and (hasattr(ParseException, 'explain') and
+                    callable(getattr(ParseException, 'explain'))):
+            print(ParseException.explain(err))
+        else:
+            print(
+                err.line + "\n" +
+                " "*(err.column-1) + "^\n" +
+                str(err)
```

Comment:
Isn't it better to use format-strings (or `logging` does that).

Reply:
Following our discussion in https://github.com/pydot/pydot/issues/171#issuecomment-622489506, I will use logging instead. I will use `%`-string formatting with the variables [passed as arguments](https://docs.python.org/3.8/howto/logging.html#logging-variable-data) to the logger to [defer actual formatting](https://stackoverflow.com/questions/5082452/string-formatting-vs-format/5082809#5082809).